block-node/verification/src/main/java/org/hiero/block/node/verification/session/impl/ExtendedMerkleTreeSession.java

```diff
@@ -31,8 +30,7 @@ public class ExtendedMerkleTreeVerificationSessionV0680 implements BlockVerifica
      */
     protected final List<BlockItemUnparsed> blockItems = new ArrayList<>();
 
-    public ExtendedMerkleTreeVerificationSessionV0680(
-            final long blockNumber, final BlockSource blockSource, final String extraBytes) {
+    public ExtendedMerkleTreeSession(final long blockNumber, final BlockSource blockSource, final String extraBytes) {
```

Comment:
To your point, `ExtendedMerkleTree` isn't very telling of the approach.
This is fine but if you think of any other attribute that describe it well let's use that.
This will essentially be the base implementation for a long time.

Reply:
Agree on follow up update. Funny as the change will bring us closer to the original.